### ISSUE TITLE
airgun.session.Session: TypeError: read() missing 1 required positional argument: 'entity_name'

### ISSUE THREAD

**Issue by jhutar** [SatelliteQE/airgun#192]
Not sure if this is an AirGun issue or not, but reporting here as a start.

Test `tests.foreman.ui_airgun.test_lifecycleenvironment.test_positive_create_chain` fails:

```
    def test_positive_create_chain(session):
        """Create Content Environment in a chain
    
        :id: ed3d2c88-ef0a-4a1a-9f11-5bdb2119fc18
    
        :expectedresults: Environment is created
    
        :CaseLevel: Integration
        """
        lce_path_name = gen_string('alpha')
        lce_name = gen_string('alpha')
        with session:
            session.lifecycleenvironment.create(
                values={'name': lce_path_name}
            )
            session.lifecycleenvironment.create(
                values={'name': lce_name},
                prior_entity_name=lce_path_name,
            )
>           lce_values = session.lifecycleenvironment.read()
E           TypeError: read() missing 1 required positional argument: 'entity_name'
```

**Comment by abalakh** [comment]
This failure is addressed in SatelliteQE/robottelo#6202
The root cause is some updates to both airgun and robottelo were proposed is separate PRs, airgun PR was merged faster (6 days ago), but robotello PR got merged just few hours ago.

**Comment by jhutar** [comment]
Awesome, thank you!
